diff -rc websvn-2.0rc4-orig/include/configclass.inc websvn-2.0rc4/include/configclass.inc
*** websvn-2.0rc4-orig/include/configclass.inc	2006-05-13 00:45:46.000000000 +0200
--- websvn-2.0rc4/include/configclass.inc	2006-12-16 00:17:23.000000000 +0100
***************
*** 459,464 ****
--- 459,465 ----
     var $spaces = 8;
     var $bugtraq = false;
     var $auth = "";
+    var $cacheDir = "";
       
     var $templatePath = "./templates/Standard/";
  
***************
*** 919,924 ****
--- 920,937 ----
        return $this->templatePath;
     }
  
+    // Cache directory
+ 
+    function setCacheDir($path)
+    {
+        $this->cacheDir = $path;
+    }
+ 
+    function getCacheDir()
+    {
+       return $this->cacheDir;
+    }
+ 
     // parentPath
     //
     // Automatically set up the repositories based on a parent path
diff -rc websvn-2.0rc4-orig/rss.php websvn-2.0rc4/rss.php
*** websvn-2.0rc4-orig/rss.php	2006-05-13 00:42:10.000000000 +0200
--- websvn-2.0rc4/rss.php	2006-12-16 00:18:45.000000000 +0100
***************
*** 72,78 ****
  
  // Cachename reflecting full path to and rev for rssfeed. Must end with xml to work
  $cachename = strtr(getFullURL($listurl), ":/\\?", "____");
! $cachename = $locwebsvnreal.DIRECTORY_SEPARATOR."cache".DIRECTORY_SEPARATOR.$cachename.@$_REQUEST["rev"]."_rssfeed.xml";
  
  $rss = new UniversalFeedCreator();
  $rss->useCached("RSS2.0", $cachename);
--- 72,80 ----
  
  // Cachename reflecting full path to and rev for rssfeed. Must end with xml to work
  $cachename = strtr(getFullURL($listurl), ":/\\?", "____");
! $cacheDir = $config->getCacheDir();
! if ($cacheDir == "") $cacheDir = $locwebsvnreal.DIRECTORY_SEPARATOR."cache";
! $cachename = $cacheDir.DIRECTORY_SEPARATOR.$cachename.@$_REQUEST["rev"]."_rssfeed.xml";
  
  $rss = new UniversalFeedCreator();
  $rss->useCached("RSS2.0", $cachename);
