diff -rc websvn-2.0-orig/include/configclass.php websvn-2.0/include/configclass.php
*** websvn-2.0-orig/include/configclass.php	2007-08-13 08:51:09.000000000 +0200
--- websvn-2.0/include/configclass.php	2008-02-05 16:26:01.000000000 +0100
***************
*** 505,510 ****
--- 505,511 ----
     var $spaces = 8;
     var $bugtraq = false;
     var $auth = "";
+    var $cacheDir = "";
       
     var $templatePath = "./templates/Standard/";
  
***************
*** 1251,1255 ****
--- 1252,1269 ----
     }
  
     // }}}
+ 
+    // Cache directory
+ 
+    function setCacheDir($path)
+    {
+        $this->cacheDir = $path;
+    }
+ 
+    function getCacheDir()
+    {
+       return $this->cacheDir;
+    }
+    
  }
  ?>
diff -rc websvn-2.0-orig/rss.php websvn-2.0/rss.php
*** websvn-2.0-orig/rss.php	2007-05-02 08:47:03.000000000 +0200
--- websvn-2.0/rss.php	2008-02-05 16:24:19.000000000 +0100
***************
*** 67,73 ****
  
  // Cachename reflecting full path to and rev for rssfeed. Must end with xml to work
  $cachename = strtr(getFullURL($listurl), ":/\\?", "____");
! $cachename = $locwebsvnreal.DIRECTORY_SEPARATOR."cache".DIRECTORY_SEPARATOR.$cachename.@$_REQUEST["rev"]."_rssfeed.xml";
  
  $rss = new UniversalFeedCreator();
  $rss->useCached("RSS2.0", $cachename);
--- 67,75 ----
  
  // Cachename reflecting full path to and rev for rssfeed. Must end with xml to work
  $cachename = strtr(getFullURL($listurl), ":/\\?", "____");
! $cacheDir = $config->getCacheDir();
! if ($cacheDir == "") $cacheDir = $locwebsvnreal.DIRECTORY_SEPARATOR."cache";
! $cachename = $cacheDir.DIRECTORY_SEPARATOR.$cachename.@$_REQUEST["rev"]."_rssfeed.xml";
  
  $rss = new UniversalFeedCreator();
  $rss->useCached("RSS2.0", $cachename);
