--- original/View.pm	2004-12-04 14:22:51.000000000 +0100
+++ updated/View.pm	2004-12-04 14:23:06.000000000 +0100
@@ -212,17 +212,14 @@
     $breakRev = $maxrev - $TWiki::numberOfRevisions + 1;
   }
   while( $i > 0 ) {
-    if( $i == $rev) {
-      $revisions = "$revisions | r1.$i";
-    } else {
-      $revisions = "$revisions | <a href=\"%SCRIPTURLPATH%/view%SCRIPTSUFFIX%/%WEB%/%TOPIC%?rev=1.$i\">r1.$i</a>";
-    }
+    $revisions = "$revisions <li class=\"revision\"><a href=\"%SCRIPTURLPATH%/view%SCRIPTSUFFIX%/%WEB%/%TOPIC%?rev=1.$i\">Rev $i</a>";
+
     if( $i != 1 ) {
+      $j = $i - 1;
+      $revisions = "$revisions <a href=\"%SCRIPTURLPATH%/rdiff%SCRIPTSUFFIX%/%WEB%/%TOPIC%?rev1=1.$i&amp;rev2=1.$j\">(diff $j)</a></li>";
+
       if( $i == $breakRev ) {
         $i = 1;
-      } else {
-        $j = $i - 1;
-        $revisions = "$revisions | <a href=\"%SCRIPTURLPATH%/rdiff%SCRIPTSUFFIX%/%WEB%/%TOPIC%?rev1=1.$i&amp;rev2=1.$j\">&gt;</a>";
       }
     }
     $i = $i - 1;
